h/static/scripts/widget-controller.js

```diff
@@ -97,8 +117,19 @@ module.exports = function WidgetController(
     visibleThreads.setRootThread(thread());
     $scope.selectedTab = annotationUI.getState().selectedTab;
 
+    if (countWaitingToAnchorAnnotations(annotationUI.getState().annotations) > 0) {
+      $scope.waitingToAnchorAnnotations = true;
+    } else {
+      $scope.waitingToAnchorAnnotations = false;
+    }
+
     $scope.totalAnnotations = countAnnotations(annotationUI.getState().annotations);
     $scope.totalNotes = countNotes(annotationUI.getState().annotations);
+    $scope.totalOrphans = countOrphans(annotationUI.getState().annotations);
+
+    if (!features.flagEnabled('orphans_tab')) {
+      $scope.totalAnnotations = $scope.totalAnnotations + $scope.totalOrphans;
```

Comment:
It is a bit confusing here that `$scope.totalAnnotations` is assigned twice if the flag is set.

I think this would be clearer as:

```
if (!features.flagEnabled('orphans_tab')) {
  $scope.totalAnnotations = ...
} else {
  $scope.totalAnnotations = ...
}
```